package.json

```diff
@@ -18,6 +19,7 @@
     "lint": "jshint js preprocessing validation",
     "jasmine-tests": "jasmine",
     "test": "npm run lint && npm run jasmine-tests && npm run validate",
-    "validate": "node validation/markets-validator.js"
+    "validate": "node validation/markets-validator.js",
+    "start": "http-server -f index.html"
```

Comment:
Well we do state in the readme how to run a local server (which is also the place where people are going to look for it) so I don't think being more verbose in the commit message here really is any help.

And I think people are quite used to follow the instructions for how to run a projects locally and do not just assume that a simple `python3 -m http.server` is going to work.